modules/speech-to-text-workflowoperation/src/main/java/org/opencastproject/workflow/handler/speechtotext/SpeechToTextWorkflowOperationHandler.java

```diff
@@ -158,18 +165,19 @@ public WorkflowOperationResult start(WorkflowInstance workflowInstance, JobConte
    * @param parentMediaPackage The media package where the track is located.
    * @param tagsAndFlavors Tags and flavors instance (to get target flavor information)
    * @param appendSubtitleAs Tells how the subtitles file has to be appended.
+   * @param translate Allows
```

Comment:
Missed the rest of the description here.

Reply:
Done!